install/servlet/docs/index.html

```diff
@@ -38,30 +38,30 @@
 </head>
 <body>
 <div align="center">
-<h1>Welcome to the Jakarta<sup><small></small></sup> Servlet TCK, Version 4.0<br>
+<h1>Welcome to the Jakarta<sup><small></small></sup> Servlet TCK, Version 5.0<br>
 <span class="emphasize">Your Starting Point</span></h1>
 </div>
 
 <hr>
-<h2>Guide to Jakarta Servlet 4.0 TCK Documentation</h2>
-<p>The Jakarta Servlet Technology Compatibility Kit (TCK) 4.0 documentation includes the following:</p>
+<h2>Guide to Jakarta Servlet 5.0 TCK Documentation</h2>
+<p>The Jakarta Servlet Technology Compatibility Kit (TCK) 5.0 documentation includes the following:</p>
 <ul>
 	<li>
-	  <p>The <a href="ReleaseNotes-servlet.html">Jakarta</a><a href="javatest/javatest.pdf"><em><small><sup></sup></small></em></a><a href="ReleaseNotes-servlet.html"> Servlet TCK 4.0 Release Notes</a> provide the information that you need to install, set up, and run the Jakarta Servlet TCK.</p>
+	  <p>The <a href="ReleaseNotes-servlet.html">Jakarta</a><a href="javatest/javatest.pdf"><em><small><sup></sup></small></em></a><a href="ReleaseNotes-servlet.html"> Servlet TCK 5.0 Release Notes</a> provide the information that you need to install, set up, and run the Jakarta Servlet TCK.</p>
 	</li>
 	<li>
-	  <p>The <em>Jakarta Servlet Technology Compatibility Kit 4.0 User's Guide</em> provides the information you need to install, configure, and run the Jakarta Servlet TCK. The User's Guide also contains the rules you need to pass for certification.</p>
+	  <p>The <em>Jakarta Servlet Technology Compatibility Kit 5.0 User's Guide</em> provides the information you need to install, configure, and run the Jakarta Servlet TCK. The User's Guide also contains the rules you need to pass for certification.</p>
 	</li>
 	<li>
-	  <p>The <a href="assertions/api-assertions.html">Javadoc Assertion List</a> lists all the javadoc assertions that are tested by the Jakarta Servlet TCK.</p></li>
+	  <p>The <a href="assertions/ServletSpecAssertions.html">Javadoc Assertion List</a> lists all the javadoc assertions that are tested by the Jakarta Servlet TCK.</p></li>
```

Comment:
Did you mean ServletJavadocAssertions.html here

Reply:
Good catch, I'll correct them!